fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -530,8 +530,19 @@ Future<Void> managementClusterCheckEmpty(Transaction tr) {
 	return Void();
 }
 
+ACTOR template <class Transaction>
+Future<TenantMode> getManagementClusterTenantMode(Transaction tr) {
```

Comment:
Although we are using it on the management cluster, this function is sufficiently generic that it could be named without referring to the management cluster. I think it's also worth adding "Configured" because the functional tenant mode of a cluster is not necessarily the same as what this value says because of the metacluster overrides.

```suggestion
Future<TenantMode> getClusterConfiguredTenantMode(Transaction tr) {
```